### DISCUSSION TITLE
Tab order

### DISCUSSION THREAD

**Discussion by vbjay** [gitextensions/gitextensions#9535] [General]
Hoping https://developercommunity2.visualstudio.com/t/Tab-Order-not-working/1063906#T-N1514070
 will get this feature in visual studio working soon.  It's real annoying trying to set tabindex with the propertygrid when they have a tool that does it visually and yet doesn't work in net 5.0.

**Comment by RussKie** [comment]
I'm afraid I can't give you any definitive dates, apart from "hopefully soon".
